packages/geo-diary/src/components/ui/appointment-time/appointment-time.tsx

```diff
@@ -21,29 +21,34 @@ export const AppointmentTime = () => {
   const { appState, setAppState } = useAppState()
   const { time } = appState
   return (
-    <ButtonGroup className="is-narrow mb-2" isCentered>
-      <Button
-        type="button"
-        variant={time !== 'TOMORROW' && time !== 'WEEK' ? 'primary' : 'secondary'}
-        onClick={handleChangeTime({ setAppState, time: 'TODAY' })}
-      >
-        TODAY
-      </Button>
-      <Button
-        type="button"
-        variant={time === 'TOMORROW' ? 'primary' : 'secondary'}
-        onClick={handleChangeTime({ setAppState, time: 'TOMORROW' })}
-      >
-        TOMORROW
-      </Button>
-      <Button
-        type="button"
-        variant={time === 'WEEK' ? 'primary' : 'secondary'}
-        onClick={handleChangeTime({ setAppState, time: 'WEEK' })}
-      >
-        WEEK
-      </Button>
-    </ButtonGroup>
+    <div className={elMb2}>
+      <FlexContainer isFlexJustifyCenter>
+        <ToggleRadio
+          name="appointment-time"
+          onChange={handleChangeTime({ setAppState })}
+          options={[
+            {
+              id: 'today',
+              value: 'TODAY',
+              text: 'Today',
+              isChecked: time === 'TODAY',
+            },
+            {
+              id: 'tomorrow',
+              value: 'TOMORROW',
+              text: 'Tomorrow',
+              isChecked: time === 'TOMORROW',
+            },
+            {
+              id: 'week',
+              value: 'WEEK',
+              text: 'Week',
+              isChecked: time === 'WEEK',
```

Comment:
use enum for time values? Easier for type hinting and usage

```ts
enum AppTimeRange {
  TODAY = 'today',
  TOMORROM = 'tomorrow',
  WEEK = 'week',
}

const options: {
  id: keyof AppTimeRange,
  value: AppTimeRange,
  text: string,
  isChecked: boolean
}[] = Object.keys(AppTimeRange).map(timeRange => ({  id: timeRange, value: AppTimeRange[timeRange], text: timeRange.capitalize(), isChecked: time === AppTimeRange[timeRange] })
```

types vs enum: types such as `type AppTimeRange = 'TODAY' | 'TOMORROW'` cannot be used as a value. Where as an enum can be a type and a value.

Reply:
As discussed, an enum may not be appropriate here and would be a big refactor